yaya_base/shiori3.dic

```diff
@@ -499,6 +475,25 @@ request
 		}
 	}
 
+	if SHIORI3FW.Eventid == 'OnAiTalk' || SHIORI3FW.Eventid == 'OnAITalk' {
+		SHIORI3FW.Eventid = 'OnAITalkNewEvent'
+	}
+
+	if TOLOWER(SHIORI3FW.SecurityLevel) == 'external'{
+		SHIORI3FW.Eventid = "ExternalEvent." + SHIORI3FW.Eventid
```

Comment:
use single quote: 
`SHIORI3FW.Eventid = 'ExternalEvent.' + SHIORI3FW.Eventid`

Reply:
ohhhh
thanks for check!
fixed at https://github.com/YAYA-shiori/yaya-dic/commit/910e5b67905afc522a19135985982cf1edc1b763